build.gradle.kts

```diff
@@ -1,17 +1,9 @@
 plugins {
+    id("com.android.application") version "8.0.1" apply false
+    id("com.android.library") version "8.0.1" apply false
```

Comment:
Is this line necessary for an application?

Reply:
It seams not, even if whole Android Developer documentation suggests it. Fixed.